.circleci/config.yml

```diff
@@ -697,3 +770,119 @@ workflows:
+          requires:
+            - generate-random-aws-credentials
+            - preflight
+      - unit-tests:
+          requires:
+            - generate-random-aws-credentials
+            - preflight
+      - docker-build:
+          name: docker-build-amd64
+          platform: amd64
+          machine_image: << pipeline.parameters.ubuntu-amd64-machine-image >>
+          resource_class: medium
+          requires:
+            - generate-random-aws-credentials
+            - preflight
+      - docker-build:
+          name: docker-build-arm64
+          platform: arm64
+          # The latest version of ubuntu is not yet supported for ARM:
+          # https://circleci.com/docs/2.0/arm-resources/
+          machine_image: << pipeline.parameters.ubuntu-arm64-machine-image >>
+          resource_class: arm.medium
+          requires:
+            - generate-random-aws-credentials
+            - preflight
+      - docker-test:
+          name: docker-test-arm64
+          platform: arm64
+          resource_class: arm.medium
+          machine_image: << pipeline.parameters.ubuntu-arm64-machine-image >>
+          requires:
+            - generate-random-aws-credentials
+            - docker-build-arm64
+      - docker-test:
+          name: docker-test-amd64
+          platform: amd64
+          resource_class: medium
+          machine_image: << pipeline.parameters.ubuntu-amd64-machine-image >>
+          requires:
+            - generate-random-aws-credentials
+            - docker-build-amd64
+      - bootstrap-tests:
+          requires:
+            - generate-random-aws-credentials
+            - docker-build-amd64
+      - capture-not-implemented:
+          name: collect-not-implemented
+          requires:
+            - generate-random-aws-credentials
+            - docker-build-amd64
+      - report:
+          requires:
+            - generate-random-aws-credentials
+            - itest-sfn-legacy-provider
+            - itest-s3-v2-legacy-provider
+            - itest-cloudwatch-v2-provider
+            - acceptance-tests
+            - docker-test-amd64
+            - docker-test-arm64
+            - collect-not-implemented
+            - unit-tests
+      - send-stats:
+          filters:
+            branches:
+              only: master
+          requires:
+            - itest-sfn-legacy-provider
+            - itest-s3-v2-legacy-provider
+            - itest-cloudwatch-v2-provider
+            - acceptance-tests
+            - docker-test-amd64
+            - docker-test-arm64
+            - collect-not-implemented
+            - unit-tests
+      - push:
+          filters:
+            branches:
+              only: master
+          requires:
+            - itest-sfn-legacy-provider
+            - itest-s3-v2-legacy-provider
+            - itest-cloudwatch-v2-provider
+            - acceptance-tests
+            - docker-test-amd64
+            - docker-test-arm64
+            - unit-tests
```

Comment:
I think it would be much simpler to:
- Introduce a boolean parameter which enables or disables the account ID and region randomization.
  - This parameter is `false` by default, but can be enabled when triggered manually or in a schedule trigger.
- The schedule trigger then just sets the parameter to `true`.

This would have the following benefits:
- The whole pipeline definition (which is duplicated here) could be reused.
- A user of the pipeline would be able to also manually trigger the pipeline run with enabled account ID  and region randomization.

What's your opinion? Is there a reason you explicitly defined a second workflow here?
I'm happy to assist if you need any help in migrating the current approach :)

Reply:
As discussed we will configure this through circlici project settings